ocean_provider/run.py

```diff
@@ -46,7 +46,8 @@ def handle_error(error):
     response.status_code = code
     response.headers["Connection"] = "close"
 
-    logger.error(f"error: {error}, payload: {request.data}", exc_info=1)
+    if code != 404:
+        logger.error(f"error: {error}, payload: {request.data}", exc_info=1)
```

Comment:
> Alternately I can log it as info, if needed.
Yeah, I think we should still log 404 responses. Personally, I see no problem logging them as errors, but I'm okay with logging them as info instead.